test/unit/mdc-menu/simple.foundation.test.js

```diff
@@ -106,19 +106,19 @@ testFoundation('#open adds the open class to the menu', ({foundation, mockAdapte
 
 testFoundation('#open removes the animation class at the end of the animation',
     ({foundation, mockAdapter, mockRaf}) => {
-  td.when(mockAdapter.getAccurateTime()).thenReturn(0);
-  td.when(mockAdapter.hasClass('mdc-simple-menu--open-from-top-right')).thenReturn(true);
+      td.when(mockAdapter.getAccurateTime()).thenReturn(0);
```

Comment:
Honestly, I care less about how it specifically gets indented, and more that our codebase is completely consistent throughout. One could always write the tests however they wanted and then run `npm run fix:js` ya know?